src/App.vue

```diff
@@ -34,47 +29,51 @@ function sleep(ms: number) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
-const widgets = [
-  { name: 'WidgetWeather', component: WidgetWeather },
-  { name: 'WidgetNews', component: WidgetNews },
-  { name: 'WidgetAtCoder', component: WidgetAtCoder },
-  { name: 'WidgetCalendar', component: WidgetCalendar },
-  { name: 'WidgetClock', component: WidgetClock }
+let widgets = [
```

Comment:
```suggestion
const widgets = [
```
ごめんこれ要素数は増減しないのでconstでよさそう